docs/en/seatunnel-engine/logging.md

```diff
@@ -80,6 +80,28 @@ appender.file.layout.pattern = [%X{ST-JID}] %d{yyyy-MM-dd HH:mm:ss,SSS} %-5p [%-
 
 SeaTunnel Engine automatically integrates Log framework bridge, allowing existing applications that work against Log4j1/Logback classes to continue working.
 
+### SeaTunnel Log Configuration
+
+### Scheduled Deletion of Old Logs
+
+SeaTunnel supports scheduled deletion of old log files to prevent disk space exhaustion. You can add the following configuration in the `log4j2.properties` file:
```

Comment:
why `log4j2.properties`？

Reply:
clerical error. I still working on it. will fix it together